config/locales/client.en.yml

```diff
@@ -1937,6 +1938,7 @@ en:
         common: "That password is too common."
         same_as_username: "Your password is the same as your username."
         same_as_email: "Your password is the same as your email."
+        server_message: "Your password %{message}"
```

Comment:
We can't concat the server's message into a sentence structure on the client side because the way sentences are structured differs between locales. It might work for english but may not work for other locales. Also, this makes it hard for translator to translate because they don't have context of what `message` can be.

Reply:
> Is there a way for the server to give a full formed message for AR validations (that you know of) instead of these ones where it leaves the attribute off? I can look too...

Yup if you do `user.errors.full_messages`, it'll prepend the attribute's name but I don't think that will work for us since we will have to translate the attribute's name for all the different locales too. 

I think the easiest way around this problem is to look at the key of the error  on the server side and then return the full custom error message string?

Reply:
Okay will try that out. Kind of annoying because then we are duplicating messages, but not much else we can do. It would have been better to do that from the start but this ship has sailed :sweat_smile:

Reply:
Actually we are already doing this for other attributes so I'm doing it for password too and using full_messages